src/utils/link-interceptor.js

```diff
@@ -25,6 +25,11 @@ export const interceptLinks = event => {
     return true;
   }
 
+  const newTab = event.ctrlKey || event.metaKey;
+  if (newTab) {
+    return true; // will open in new tab b/c we dont eat the event
```

Comment:
question: does returning true/false matter here? Its just an onClick handler for a div and I can't tell that anything responds to it...